package.json

```diff
@@ -36,11 +36,12 @@
     "@redhat-cloud-services/frontend-components-charts": "^3.2.6",
     "@redhat-cloud-services/frontend-components-notifications": "^4.1.0",
     "@redhat-cloud-services/frontend-components-translations": "^3.2.7",
-    "@redhat-cloud-services/frontend-components-utilities": "^4.0.7",
+    "@redhat-cloud-services/frontend-components-utilities": "^4.0.9",
     "@reduxjs/toolkit": "^2.2.0",
     "@unleash/proxy-client-react": "^4.1.2",
     "axios": "^1.6.7",
     "lodash": "^4.17.21",
+    "p-all": "^5.0.0",
```

Comment:
Is this necessary? I don't see any imports of this module or any calls for the functions from this module.
I removed it while testing and I don't see any issues.

Reply:
Thanks, forgot about it